### DISCUSSION TITLE
1.2.0

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#23] [Announcements]
1. 新增个人影院, 对接的jellyfin数据。需要先安装jellyfin插件并注册账户和创建媒体库, 通过proxy转发jellyfin接口
2. 优化 终端、文件管理的弹窗 UI
3. 应用商店显示插件数量
4. 去除 quickstar 的硬依赖，不安装 quickstar 也可以使用了，但是目前还有部分功能需要调用 quckstar 的接口 （网络配置，系统信息，docker开关）

**Comment by Fengerxuanxiao** [comment]
文件管理的弹窗 UI，有点掉帧卡顿

**Reply by Carseason** [reply]
文件弹窗用的iframe导致的，未来换成内部组件会好很多

**Reply by Carseason** [reply]
https://github.com/Carseason/routerdog/discussions/27
